### PR TITLE
Add country_enumable concern

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Review by exu3 — APPROVED** [review]
Nice, thanks Max!

File: app/models/partnered_signup.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@ class PartneredSignup < ApplicationRecord
   include PublicIdentifiable
   set_public_id_prefix :sup
 
+  include CountryEnumable
+  has_country_enum :country
```
Comment (garyhtou) [inline app/models/partnered_signup.rb]: Should this be:
```suggestion
  has_country_enum :owner_address_country
```

**Comment by maxwofford** [comment]
Thanks for catching that @garyhtou! The previous code technically works, but we should probably update it from being a string to an int.

File: app/models/partnered_signup.rb (hunk 1 above)
Comment (exu3) [reply app/models/partnered_signup.rb]: Ah thanks!

**Comment by garyhtou** [comment]
yeah np! i think it might already be an int
image

**Comment by maxwofford** [comment]
In that case, is this ok to merge @garyhtou @exu3?

**Comment by garyhtou** [comment]
go for it!

**Pull request merged**